pkg/admission/admit_job.go

```diff
@@ -140,10 +140,16 @@ func validateJob(job v1alpha1.Job, reviewResponse *v1beta1.AdmissionResponse) st
 	// invalid job plugins
 	if len(job.Spec.Plugins) != 0 {
 		for name := range job.Spec.Plugins {
-			if _, found := plugins.GetPluginBuilder(name); !found {
+			if P := plugins.GetPluginBuilder(name); P == nil {
 				msg = msg + fmt.Sprintf(" unable to find job plugin: %s", name)
 			}
 		}
+
+		if _, found := job.Spec.Plugins["mpi"]; found {
```

Comment:
we should not check specific plugin here.

Reply:
Yeah, but we need to prevent this case, otherwise there may be unexpected conflict.

Reply:
which kind of conflict?

Reply:
Think of that we specify ssh and mpi at the same time

Reply:
find some way to do that; we should not highlight plugin name here.